packages/generate/test/testRunner.ts

```diff
@@ -26,11 +29,37 @@ import {
 
   const managementConn = await createClient(config).ensureConnected();
 
+  const version = await managementConn.queryRequiredSingle<EdgeDBVersion>(
+    `select sys::get_version()`
+  );
+
   try {
-    await applyMigrations(config);
-    // await generateQB(config);
-    // await generateQueries(config);
-    await runTests(config);
+    await applyMigrations(config, {
+      flags:
+        version.major < 3
+          ? [
+              "--to-revision",
+              "m1sxhoqfjqn7vtpmatzanmwydtxndf3jlf33npkblmya42fx3bcdoa",
+            ]
+          : undefined,
+    });
+    console.log(`\nRunning tests...`);
+    await runCommand(
+      "yarn",
+      [
+        "test:ts",
+        ...(version.major < 3
+          ? [
+              `--testPathIgnorePatterns="${[
+                "pgvector",
```

Comment:
Love this! We might even be able to throw flags into the test name to get this less specific, but this works great.